.github/workflows/mypy.yml

```
@@ -0,0 +1,22 @@
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.13
+          architecture: x64
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install mypy
+        run: pip install mypy
+      - name: Test
+        run: mypy ${{ github.event.repository.name }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)